.github/workflows/enforce-branch-name-rules.yml

```
@@ -0,0 +1,21 @@
+name: Enforce Branch Name Rules
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  enforce-branch-rules:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check branch name
+        run: |
+          echo "Source branch: ${{ github.head_ref }}"
+          if [[ "${{ github.head_ref }}" != feature/* && \
+                "${{ github.head_ref }}" != bugfix/* && \
```

```suggestion
                "${{ github.head_ref }}" != bugfix/* && \
                "${{ github.head_ref }}" != dependabot/* && \
```

Ref: https://github.com/RedisInsight/RedisInsight/pull/4535